src/ucp/wireup/select.c

```diff
@@ -1120,19 +1176,22 @@ ucp_wireup_add_rma_bw_lanes(ucp_wireup_select_ctx_t *select_ctx)
             continue;
         }
 
-        ucp_wireup_add_bw_lanes(select_ctx, &bw_info,
-                                context->mem_type_access_tls[mem_type]);
+        ucp_wireup_add_bw_lanes(select_params, &bw_info,
+                                context->mem_type_access_tls[mem_type],
+                                select_ctx);
     }
 
     return UCS_OK;
 }
 
 /* Lane for transport offloaded tag interface */
 static ucs_status_t
-ucp_wireup_add_tag_lane(ucp_wireup_select_ctx_t *select_ctx,
-                        ucp_err_handling_mode_t err_mode)
+ucp_wireup_add_tag_lane(const ucp_wireup_select_params_t *select_params,
```

Comment:
in this func we have `select_params`, `select_info` and `select_context`, which is a bit confusing. 
better keep `am_info` inside `params`

Reply:
I would like to keep `params` as a const input parameters everywhere, `ucp_wireup_select_info_t am_info`, which is a result of AM selection logic, is needed only for TM fallback. maybe renaming `am_info` to `fallback_am_info` would make it a bit clearer, WDYT?

Reply:
@brminich can you please reply?